src/components/SIngleArticle.jsx

```diff
@@ -1,25 +1,80 @@
-import axios from "axios"
-import { useEffect, useState } from "react"
-import { useParams } from "react-router-dom"
+import axios, { Axios } from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
 
-function SingleArticle(){
-    let {article_id} = useParams()
-    const [article,setArticle] = useState({})
+function SingleArticle() {
+  let { article_id } = useParams();
+  const [article, setArticle] = useState({});
+  const [comments, setComments] = useState([]);
 
-    useEffect(() => {
-        axios.get(`https://ncn-network.onrender.com/api/articles/${article_id}`)
-        .then((response) => {
-            console.log(response.data)
-            setArticle(response.data)
-        })
-    },[])
-console.log(article,'outside the effect')
+  useEffect(() => {
+    axios
+      .get(`https://ncn-network.onrender.com/api/articles/${article_id}`)
+      .then((response) => {
+        setArticle(response.data);
+      });
+  }, [article_id]);
 
-return(
-    <div className="singleArticle">
+  useEffect(() => {
+    axios
+      .get(
+        `https://ncn-network.onrender.com/api/articles/${article_id}/comments`
+      )
+      .then((response) => {
+        setComments(response.data);
+      });
+  }, [article_id]);
+
```

Comment:
Great 👍 You may want to do your axios requests in a separate file to keep things clean though :)